components/match2/commons/match_group_display_matchlist.lua

```diff
@@ -185,15 +186,22 @@ end
 
 MatchlistDisplay.propTypes.Title = {
 	title = 'string',
+	vod = 'string?',
 }
 
 --[[
 Display component for a title in a matchlist.
 ]]
 function MatchlistDisplay.Title(props)
+	local VodLink = require('Module:VodLink')
+
 	DisplayUtil.assertPropTypes(props, MatchlistDisplay.propTypes.Title)
 	local titleNode = mw.html.create('div'):addClass('brkts-matchlist-title')
 		:wikitext(props.title)
+		:node(props.vod and VodLink.display{vod = props.vod}
+			:css('position', 'absolute')
+			:css('right', '52px')
```

Comment:
probably better to move it to https://liquipedia.net/commons/MediaWiki:Common.css/Brackets.css
and also maybe increase the pright margin/padding a bit to have some spce between show/hide button and the vod link

Reply:
checking the behavior for long titles it def needs some adjustments if you want to add the vod icon

![B933DDA7-99FE-400E-AF9A-72E04DACC757](https://user-images.githubusercontent.com/75081997/161201398-e2d5e117-8115-4969-862e-943bebbec46b.jpeg)

also i suggest to push it a few px more to the left so that between the icon and the show button is a bit more space

Reply:
Should indeed reserve some space if set so that the title doesn't go under, yes.